lexicons/app/bsky/feed/post.json

```diff
@@ -38,6 +38,10 @@
             "type": "union",
             "refs": ["com.atproto.label.defs#selfLabels"]
           },
+          "tags": {
+            "type": "array",
+            "items": { "type": "string", "maxLength": 640, "maxGraphemes": 64 }
```

Comment:
at some point we might want a string format like "slug" or "keyword" which has some generic character restrictions. that's a larger protocol tangent and we don't have that today and it shouldn't block this work.